packages/babel-helpers/src/helpers.js

```diff
@@ -449,7 +449,14 @@ helpers.interopRequireWildcard = defineHelper(`
       var newObj = {};
       if (obj != null) {
         for (var key in obj) {
-          if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key];
+          if (Object.prototype.hasOwnProperty.call(obj, key)) {
+            var desc = Object.getOwnPropertyDescriptor(obj, key);
+            if (desc.get || desc.set) {
+              Object.defineProperty(newObj, key, desc);
```

Comment:
I think if we're going to do this, we should check for the existence of `Object.defineProperty`, or we should finally expose a flag for modules to opt in or out of `Object.defineProperty` usage. Otherwise this is going to make things unusable for people targeting old environments.

Reply:
I will add a check for `Object.defineProperty`, which I think should be enough, because when `Object.defineProperty` is not there then getters and setters are also not supported. 

It is basically supported in all environments besides IE8 and older. But IE8 is also over 10years old now. 🕵️‍♀️
What if we change in babel 7 and state that generated code from babel needs at least ES5, like react does now (they even require Map and Set). Would be nice to have statistics to see what environments people support, but I guess that does not really exist. Or did we have already complains about this?

Reply:
And core-js also has a polyfill for it, so we could also link people there.